_includes/head.html

```diff
@@ -13,4 +13,8 @@
   {% if jekyll.environment == 'production' and site.google_analytics %}
   {% include google-analytics.html %}
   {% endif %}
+
+  {% if site.gems contains 'jekyll-seo-tag' %}
+  {% include seo.html %}
```

Comment:
> That won't work because liquid doesn't lazy evaluate

OK. Agreed.
But that doesn't explain the need of having `{% seo title=false %}` by default.

Also, this solution wont work in the edge-situation where the user has the plugin loaded only via the `:jekyll_plugins`